backend/src/services/PhoneNumberValidation.ts

```diff
@@ -65,10 +72,32 @@ function PhoneNumberValidationService(
     });
   }
 
+  async function invalidateExpiredEntries() {
+    const expiryDate: Date = DateTime.now().minus(VALIDITY_LENGTH).toJSDate();
+    const expiredEntries = await PhoneNumberValidationModel.findAll({
+      where: {
+        updatedAt: {
+          // Less than because the expired entries are earlier than our cutoff.
+          [Op.lt]: expiryDate,
+        },
+      },
+    });
+
+    // Expire these in batches
+    await Promise.all(
+      expiredEntries
+        .slice(0, EXPIRY_BATCH_SIZE)
```

Comment:
would this just operate on the first 100?

Reply:
Yes, the intent is to only operate on the first 100. Considering that there is a backlog of expired entries, I think that if we try to do everything at once, we'll flood the app with too much work.